styles/styles.css

```diff
@@ -0,0 +1,270 @@
+* {
+  box-sizing: border-box;
+}
+button, input, select {
+  font-family: inherit;
+  font-size: 100%;
+}
+header {
+  background-image: url(../images/header-image.jpg);
+  background-size: cover;
+}
+footer {
+  color: white;
+  background-color: rgb(13, 45, 78);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 3rem 0 3rem;
+}
+body {
+  margin: auto;
+  font-size: 16px;
+}
+main {
+  max-width: 1440px;
+}
+.logo {
+  padding: 3rem 0rem 0rem 6rem;
+  width: 100%;
+  min-width: 320px;
+}
+.tagline {
+  font-size: 1.55rem;
+  margin-top: -1rem;
```

Comment:
be careful with negative margin, try to avoid it. You might be able to do something with line-height to move the text closer to the image if that's what you're after.